osu.Game/Overlays/SettingsOverlay.cs

```diff
@@ -68,6 +68,9 @@ public SettingsOverlay()
         public void ShowAtControl<T>()
             where T : Drawable
         {
+            // if search isn't cleared then the target control won't be visible if it doesn't match the query
+            SearchTextBox.Current.Value = "";
```

Comment:
```suggestion
            SearchTextBox.Current.SetDefault();
```

is a bit nicer/more idiomatic